.github/workflows/osv-scanner-pr.yml

```diff
@@ -0,0 +1,106 @@
+# This file is adapted from https://github.com/google/osv-scanner
+
+name: OSV-Scanner PR scanning
```

Comment:
It is not PR scanning anymore, sorry I missed this earlier.

Reply:
done